catalog/networking/network/vpn.yaml

```diff
@@ -35,17 +48,20 @@ metadata:
     cnrm.cloud.google.com/project-id: project-id # kpt-set: ${project-id}
     cnrm.cloud.google.com/blueprint: cnrm/landing-zone:networking/v0.3.0
 spec:
-  localTrafficSelector:
-    - "192.168.0.0/16"
-  peerIp: "15.0.0.120"
+  peerExternalGatewayInterface: 0
+  peerExternalGatewayRef:
+    name: network-name-ext-vpn-gateway # kpt-set: ${network-name}-ext-vpn-gateway
   region: us-central1 # kpt-set: ${region}
+  routerRef:
+    name: network-name-router # kpt-set: ${prefix}${network-name}-router
   sharedSecret:
     valueFrom:
       secretKeyRef:
         name: vpn-shared-secret # kpt-set: ${vpn-secret-name}
         key: vpn-shared-secret # kpt-set: ${vpn-secret-key}
-  targetVPNGatewayRef:
-    name: network-name-vpn-gateway # kpt-set: ${network-name}-vpn-gateway
+  vpnGatewayInterface: 0
+  vpnGatewayRef:
+    name: network-name-ha-vpn-gateway # kpt-set: ${network-name}-ha-vpn-gateway
 ---
 apiVersion: compute.cnrm.cloud.google.com/v1beta1
 kind: ComputeAddress
```

Comment:
Do we still need a compute address for HA vpn?

Reply:
We don't, that was a miss from me.